drivers/focuser/lacerta_mfoc.cpp

```diff
@@ -445,12 +437,22 @@ IPState lacerta_mfoc::MoveAbsFocuser(uint32_t targetTicks)
 IPState lacerta_mfoc::MoveRelFocuser(FocusDirection dir, uint32_t ticks)
 {
     // Calculation of the demand absolute position
-    uint32_t targetTicks = FocusAbsPosN[0].value + (ticks * (dir == FOCUS_INWARD ? -1 : 1));
+    uint32_t targetTicks = 0;
+    if (dir == FOCUS_INWARD) {
+      targetTicks = FocusAbsPosN[0].value - ticks;
+    } else {
+      targetTicks = FocusAbsPosN[0].value + ticks;
+    }
```

Comment:
How is this any different from the line you removed above?

Reply:
The ole line worked on my 32b Linux-Notebook, but not on a 64b-Raspberry PI where it results to "-1" targetTicks. I splitted the line for better debugging options - and with this change it gives the right results. Don't know why...

Reply:
I think a safer way is this:

auto targetTicks = std::clamp(FocusAbsPosN[0].value + (ticks * (dir == FOCUS_INWARD ? -1 : 1)), FocusAbsPosN[0].min, FocusAbsPosN[0].max)

Reply:
Cool - I'll change this today evening.

Reply:
change commited - plz review again